euphonic/plot.py

```diff
@@ -20,7 +20,7 @@
 
 
 def plot_1d_to_axis(spectra: Union[Spectrum1D, Spectrum1DCollection],
-                    ax: Axes,
+                    ax: Axes, labels: Optional[List[str]] = None,
```

Comment:
I guess having `Sequence` could be confusing, because then what does a `str` mean? Is that a label for one line or N 1-character labels? Almost certainly the former, but still...

Do we want to support just having `str` for one line label? `Optional[Union[List[str], str]]`?

Reply:
It's a straight trade-off in usability vs complexity, I'd be happy either way. Users are going to insert a string occasionally.

We either keep it strict with `Optional[List[str]]` and follow that up with an explicit `isinstance(labels, list)` check to make sure users don't get creative, or we go for the permissive `Optional[Sequence[str]]` with a check to convert `str` to `[str]`. (`str` *is* a `Sequence[str]`, so no `Union` is necessary.)